src/adapters/MeetingsJSONAdapter.js

```diff
@@ -854,4 +886,61 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
 
     return concat(initialState$, settingsEvent$);
   }
+
+  /**
+   * Returns an observable that emits the display data of the video switcher control.
+   *
+   * @param {string} ID  ID of the meeting for which to update display
+   * @returns {Observable.<MeetingControlDisplay>} Observable that emits control display data
+   * @private
+   */
+  videoControl(ID) {
+    const meeting = this.datasource[ID];
+    const videoDevices = defer(() => this.getAvailableDevices('videoinput'));
+    let initialState$;
+
+    if (meeting) {
+      initialState$ = videoDevices.pipe(
+        map((devices) => ({
+          ID: SWITCH_VIDEO_DEVICE,
+          tooltip: 'Video Devices',
+          options: devices && devices.map((device) => ({
```

Comment:
videoDevices

Reply:
the linter doesn't let me. I already used this name.